src/doc/src/reference/config.md

```diff
@@ -118,6 +118,7 @@ offline = true              # do not access the network
 # Same keys as for [patch] in Cargo.toml
 
 [profile.<name>]         # Modify profile settings via config.
+inherits = "dev"         # If an option is missing, use dev's value.
```

Comment:
Could we make the comment more informative? Such as

```suggestion
inherits = "…"           # Inherits settings from the other profile
```

Reply:
I think both are equally informative, but I'll use your suggestion.